src/pages/account/login.jsx

```diff
@@ -0,0 +1,35 @@
+import { Navbar } from "../../components/all-comp"
+import { AccLinks } from "../../components/account-comp/acc-links"
+export const Login = () => {
+    return (
+        <div>
+            <div class="container product-navbar">
+                <Navbar />
```

Comment:
Try to keep Navbar outside of Routes.
As It will be present in most of the Routes.  Then you won't need to rewrite Navbar in every route.

> Not very important so you may do it later if you're on a time crunch.

Reply:
Ok, I will do it later